src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partner = sex => (
+    sex === 'm' ? 'wife' : 'husband'
+  );
```

Comment:
```suggestion
  const partner = sex === 'm' ? 'wife' : 'husband'
```

No need to create function in this case

Reply:
> No need to create function in this case

Incorrectly functions refuse to work properly.

Reply:
Nazar suggested that you can rewrite partner and use it as a variable instead of the function